inference-engine/tests/functional/plugin/gna/shared_tests_instances/behavior/ov_infer_request/io_tensor.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "behavior/ov_infer_request/io_tensor.hpp"
+
+using namespace ov::test::behavior;
+
+namespace {
+const std::vector<std::map<std::string, std::string>> configs = {
+    {{"GNA_DEVICE_MODE", "GNA_SW_FP32"},
+     {"GNA_DEVICE_MODE", "GNA_SW_EXACT"}}
+};
+
+std::vector<ov::element::Type> prcs = {
+    ov::element::boolean,
+    ov::element::bf16,
+    ov::element::f16,
+    ov::element::f32,
+    ov::element::f64,
+    ov::element::i4,
+    ov::element::i8,
+    ov::element::i16,
+    ov::element::i32,
+    ov::element::i64,
+    ov::element::u1,
+    ov::element::u4,
+    ov::element::u8,
+    ov::element::u16,
+    ov::element::u32,
+    ov::element::u64,
```

Comment:
Currently, we support only f32, i16, u8 precisions. So I'm wondering how it should work.

Reply:
shared test verify that exception will be thrown for unsupported precisions